app/Http/Controllers/Api/TilesController.php

```
@@ -0,0 +1,118 @@
+                    'path' => $layerName
+                ];
+            }
+        }
+
+        return response()->json(['layers' => $layers]);
+    }
+
+    /**
+     * Get tile layer bounds from tilemapresource.xml
+     */
+    public function getBounds(Request $request, $layer)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)